engine/src/test/java/io/zeebe/engine/processor/workflow/deployment/model/validation/ZeebeRuntimeValidationTest.java

```diff
@@ -45,7 +46,7 @@
   private static final String INVALID_PATH_QUERY_MESSAGE =
       "JSON path query is invalid: Unexpected json-path token ROOT_OBJECT";
 
-  private static final String INVALID_EXPRESSION = "?!";
+  private static final String INVALID_EXPRESSION = "=?!";
```

Comment:
The expression prefix is set by the model API builder.

```suggestion
  private static final String INVALID_EXPRESSION = "?!";
```